src/System.Private.CoreLib/src/Internal/Runtime/Augments/EnvironmentAugments.Unix.cs

```diff
@@ -30,7 +30,89 @@ private static void SetEnvironmentVariableCore(string variable, string value)
             if ("".Length != 0)
```

Comment:
This should be deleted. It has been just a marker that this is not implemented yet.

Reply:
Gotcha. Wasn't sure the point of this so I kept it. Will delete.

Reply:
Done.